torch/csrc/inductor/aoti_runtime/interface.h

```diff
@@ -112,4 +118,15 @@ AOTIRuntimeError AOTInductorModelContainerGetMaxOutputShape(
     const int64_t** ret_output_sizes,
     int64_t* ret_output_ndim);
 
+AOTIRuntimeError AOTInductorModelCreate(
```

Comment:
I would prefer to have the naming more explicitly state how it is different from AOTInductorModelContainerCreate, or at least add comments here to explain that.

Reply:
Can you suggest a name?  IMO this is pretty self explanatory so I'm struggling to come up with something satisfactory.

Reply:
Perhaps we could just add comments to explain when the user would prefer to use this function, e.g. like those sentences from the summary?